scripts/lib/mkcloud-common.sh

```diff
@@ -730,6 +730,7 @@ fi
 : ${want_mtu_size:=1500}
 # proposals:
 : ${want_designate_proposal:=0}
+: ${want_octavia_proposal:=0}
```

Comment:
minor nit: move it down 2 lines to keep the entries sorted.

Reply:
done